frontend/packages/kubevirt-plugin/integration-tests/tests/vm.non-admin.scenario.ts

```diff
@@ -38,19 +32,18 @@ const {
 
 describe('Kubevirt non-admin Flow', () => {
   const leakedResources = new Set<string>();
-  const wizard = new Wizard();
-  const configName = ProvisionConfigName.URL;
-  const provisionConfigs = getProvisionConfigs();
-  const provisionConfig = provisionConfigs.get(configName);
-  provisionConfig.networkResources = [];
-
-  const vm1Config = vmConfig(configName.toLowerCase(), testNonAdminNamespace, provisionConfig);
-  vm1Config.startOnCreation = false;
+  const vm = new VMBuilder(getBasicVMBuilder())
+    .setProvisionSource(provisionSources.URL)
+    .setDisks([rootDisk])
+    .setFlavor(flavorConfigs.Tiny)
```

Comment:
there are still some instances where specifying Tine is not required due to getBasicVM/VMT builders

Reply:
Removed all the unneccessary ones